### ISSUE TITLE
Add ability to send daily/weekly summary emails on experiment results

### ISSUE THREAD

**Issue by Auz** [growthbook/growthbook#307]

**Comment by xahhy** [comment]
We usually have some other people who don't have the GrowthBook account but just need to review the AB test report in a daily/weekly basis. Especially for a company wide spread of how things are going on in the AB test world. So it could worth to have a nice UI and data in the summary email.

**Comment by eltarotassadar** [comment]
Agreed. I've got this question from my CEO, so he dont have much time to look through platform, but he really interested to stay in touch with experiments

**Comment by lukesonnet** [comment]
We're working on summaries as part of our Slack App @Auz

**Comment by alyssamichelle** [comment]
Triage from the open-issues audit:

**Suggested action:** Fold into a GitHub Discussion

**Why:** Outbound digest vs in-app notifications. This is a product-direction ask. Please convert or fold into a Discussion rather than keeping it as a standalone issue.

- [Event webhooks](https://docs.growthbook.io/app/webhooks/event-webhooks)
- [Permissions](https://docs.growthbook.io/account/user-permissions)

**Comment by alyssamichelle** [comment]
Advised to close after review: this is now being discussed in https://github.com/growthbook/growthbook/discussions/6822 (When should GrowthBook notify you?).
